content/docs/17-glossary-all.md

```diff
@@ -16,6 +16,12 @@ import Tooltip from "shared/components/ui/Tooltip";
 # Glossary
 
 This page gives a quick reference to various object types and concepts within the Palette platform.
+## App Mode
+A mode optimized for a simpler and streamlined developer experience that allows you to only focus on the building, maintenance, testing, deployment, and monitoring of your Kubernetes applications. App Mode removes the need to worry about the infrastructure management of a Kubernetes cluster and results in a PaaS-like experience, enabling you to only focus on deploying [App Profiles](#app-profile). App Mode also allows you to deploy [Palette Virtual Clusters](#palette-virtual-cluster).
+
+## App Profile
+App Profiles are templates created with preconfigured services required for Palette Virtual Clusters deployment. App Profiles provide a way to drive consistency across virtual clusters. You can create as many profiles as required. An App Profile can be created to meet specific types of workloads on Palette Virtual Clusters.

```

Comment:
not only preconfigured but to bundle your own applications together with common necessary sercies, such as cache, db, etc, and make it easier for you to deploy those services.